lib/alexaSmartHomeV2.js

```diff
@@ -662,6 +662,13 @@ function AlexaSH2(adapter) {
                 if (callback) callback();
                 return;
             }
+
+            if (!obj.common.role || obj.common.role.indexOf('level.temperature') === -1) {
```

Comment:
why "indexof"? why not === ?

Reply:
With IndexOf, all roles that contain level.temperature are valid. In my case, the data point was automatically assigned the role level.temperature.number by the KNX adapter.

Reply:
Ok, but then startsWith should bet better.

Reply:
If I think about it again, it would be more consistent to call the getActions method and then cancel the processing based on the return "actions".

Reply:
Ok i switched to startsWith